### ISSUE TITLE
Station Planning Map

### ISSUE THREAD

**Issue by StuMackenzie** [MotusDev/Motus-TO-DO#873]
For discussion and refinement.

We've been discussing the concept of a planning map where projects can populate future deployment locations. This has become important to label stations that are using different hardware that isn't yet compatible with Motus, but they want to be incorporated.

Process: 
1) User creates project, or uses existing project, 
2) Creates a new station, but instead of entering a receiver, enters - "prospective station'
3) User creates deployment for prospective station with minimal information
4) Main Motus map is with a different colour/label. Toggle on Motus Map "See Prospective Stations"

Needed information:

Station name, lat/long, Propsective start date.
Information displayed on website - project name/contact, and prospective start date.

**Comment by leberrigan** [comment]
## TL;DR

**Important points**
- We should identify prospective locations at the **station** level, not the site level. We do this by including a new column in the 'sensorSites.stations' table which defines it's status.
- We may want to be able to decommission **sites** when we can no longer install stations at that property.
- We can populate the 'prospective stations' map with previous deployments where no active stations currently exist.

**Variables to include for each prospective location**
- **Location**: a single point, as defined when registering a new station.
- **Site**: provides landowner contact information.
- **Status**: if there is an active deployment, the station is 'active'. If there is no active deployment, it is 'prospective'. If stations can no longer be installed at that location, it is 'decommissioned'.
- **Permissions**: what are the chance's someone would be allowed to put a station here? We might want to make 3 or 4 options for this field. Unless we only want to accept sites where permission is assumed in which case this field can be ignored.
- **Mounting structure**: does a structure already exist that can be used or does one need to be erected?
- **Internet**: is internet available?
- **Power**: is AC power available?
- **Historical data**: have stations been installed here in the past?

----


Here is some information that we would want to include with each prospective station:
- **Location**: a single point, as defined when registering a new station.
- **Site**: provides landowner contact information.
- **Permissions**: what are the chances someone would be allowed to put a station here? We might want to make 3 or 4 options for this field. Unless we only want to accept sites where permission is assumed in which case this field can be ignored.
- **Mounting structure**: does a structure already exist that can be used or does one need to be erected?
- **Internet**: is internet available?
- **Power**: is AC power available?
- **Historical data**: have stations been installed here in the past?

To start, what are we going to call this tool? At first I thought it should be called **'prospective sites'** since that's what we've been talking about working with, but then again I do think it's a good idea to include prospective "station locations" within each site along with information for those locations (structure, internet, etc.). 

After that thought, it seems to make more sense to call it **'prospective stations'** and only use sites to group stations and link them to a landowner contact. This means we wouldn't be treating sites any differently from before. The stations would need a new column to describe its status as 'prospective', 'active', or 'decommissioned'. A station is 'active' as long as it has an active deployment associated with it.  

A location is 'decommissioned' when we can no longer place a station there for one reason or another. This issue here is whether we only want to decommission stations or if we need to be able to decommission entire sites.

We probably need to think about historical stations a bit more. We will have many sites where stations no longer exist but are still a good spot. These will already exist in our database so we could populate this prospective stations map from the start. We could just say 'unknown' for all the above information that we're asking about, or glean some of that information from previous installations (mounting type, landowner contact, location). 
However, when it comes to historical **site** or **stations**, we won't know whether they are still usable (i.e.; 'decommission'). That means we might not want to populate the prospective sites map using historical locations without confirming with the project PI first.

**Comment by ajmacdonald3** [comment]
The PIF Western Working Group Motus planning document uses these four categories for station status:                                                                                                                                                                                                                 R= running, B= building (funded), P = planning (proposal submitted), D= development stage	
We could add another category for essentially "would be a nice spot, but no concrete plans for development" or something along those lines, as well as the decommissioned category.

**Comment by StuMackenzie** [comment]
Discussion and development should be moved to Slack #dashboard dev or new channel.

See discussion in  - https://github.com/MotusDev/Motus-TO-DO/issues/696
